job-dsls/jobs/prod/prod_multibranch_nightly.groovy

```diff
@@ -0,0 +1,158 @@
+/**
+ * Creates jobs for multibranch-pipelines
+ * - nightly
+ * - kogito.nightly
+ * - kogito-tooling.nightly
```

Comment:
we probably can avoid this job for now, kogito-tooling is going to be dropped.
you can just comment it out if we need it later ;)

Reply:
OK - I will comment it out!